nodecg-io-midi-input/extension/index.ts

```diff
@@ -16,12 +16,34 @@ module.exports = (nodecg: NodeCG) => {
 
 class MidiService extends ServiceBundle<MidiInputServiceConfig, MidiInputServiceClient> {
     async validateConfig(config: MidiInputServiceConfig): Promise<Result<void>> {
-        new easymidi.Input(config.device).close();
+        const devices: Array<string> = new Array<string>();
+
+        easymidi.getInputs().forEach((device) => {
+            if (device.includes(config.device)) {
+                devices.push(device);
+            }
+        });
+
+        if (devices.length == 0) {
```

Comment:
```suggestion
        if (devices.length === 0) {
```
Just usual JavaScript problems